resources/metadata/detectExecuteScan.yaml

```diff
@@ -324,6 +324,17 @@ spec:
           - STEPS
         default: true
         deprecated: true
```

Comment:
Why is this deprecated?

Reply:
I don't know. I'd remove the flag if it were up to me. 😄 
Maybe someone from the BlackDuck team can answer. I see that none is assigned, though. :)

Reply:
it's just placeholder at the moment, not used value